### PR TITLE
Minor comments before workshop

### PR DESCRIPTION
**Pull request by hslzr**
Adds a few comments/suggestions

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import List from './components/List';
 
 const longestSerieComputed = (series) => {
-  
+  // Implementation goes here
```
Comment (hslzr) [inline src/App.js:6]: Might be a good idea to leave a comment here to make it both easier to search and to let the user know we're waiting for our implementation.

File: src/App.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ function App() {
   const [longestSerie, setLongestSerie] = useState("");
 
   const multiplyNumbers = () => {
-    
+    // Implementation goes here
```
Comment (hslzr) [inline src/App.js:16]: Might be a good idea to leave a comment here to make it both easier to search and to let the user know we're waiting for our implementation.

File: src/components/List.js [hunk 3]
```diff
@@ -1,19 +1,18 @@
 import React, { useState, useEffect } from 'react'
 
 const List = ({ multiplyNumbers }) => {
-    const [items, setItems] = useState([]);
+  const [items, setItems] = useState([]);
+  useEffect(() => {
+    // Implementation goes here
```
Comment (hslzr) [inline src/components/List.js:6]: Might be a good idea to leave a comment here to make it both easier to search and to let the user know we're waiting for our implementation.

File: src/App.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (hslzr) [inline src/App.js:1]: I'd say it's better not to add the hooks yet as that should also be part of the workshop: adding them in the first place.

**Review by Carlos-Gaxiola — APPROVED** [review]
Thank you for the recommendations!

**Pull request merged**